ros2topic/ros2topic/api/__init__.py

```diff
@@ -171,49 +170,10 @@ def qos_profile_from_short_keys(
         profile.durability = rclpy.qos.QoSDurabilityPolicy.get_from_short_key(durability)
     if reliability:
         profile.reliability = rclpy.qos.QoSReliabilityPolicy.get_from_short_key(reliability)
-    if depth >= 0:
+    if depth and depth >= 0:
         profile.depth = depth
     else:
         if (profile.durability == rclpy.qos.QoSDurabilityPolicy.TRANSIENT_LOCAL
                 and profile.depth == 0):
             profile.depth = 1
-
     return profile
-
-
-def add_qos_arguments_to_argument_parser(
```

Comment:
I've found an instance of this function being used in the wild. In the future, we should be mindful of removing API from this file since it appears as part of the public API (unless we otherwise state that users should't rely on it).